frontend/pages/hosts/ManageHostsPage/ManageHostsPage.tsx

```diff
@@ -1881,62 +1735,47 @@ const ManageHostsPage = ({
   return (
     <>
       <MainContent>
-        <>
-          {renderForm()}
-          {!isAddLabel && !isEditLabel && (
-            <div className={`${baseClass}`}>
-              <div className="header-wrap">
-                {renderHeader()}
-                <div className={`${baseClass} button-wrap`}>
-                  {!isSandboxMode &&
-                    canEnrollHosts &&
-                    !hasHostErrors &&
-                    !hasHostCountErrors && (
-                      <Button
-                        onClick={() => setShowEnrollSecretModal(true)}
-                        className={`${baseClass}__enroll-hosts button`}
-                        variant="inverse"
-                      >
-                        <span>Manage enroll secret</span>
-                      </Button>
-                    )}
-                  {canEnrollHosts &&
-                    !hasHostErrors &&
-                    !hasHostCountErrors &&
-                    !(
-                      getStatusSelected() === ALL_HOSTS_LABEL &&
-                      selectedLabel?.count === 0
-                    ) &&
-                    !(
-                      getStatusSelected() === ALL_HOSTS_LABEL &&
-                      filteredHostCount === 0
-                    ) && (
-                      <Button
-                        onClick={toggleAddHostsModal}
-                        className={`${baseClass}__add-hosts button button--brand`}
-                      >
-                        <span>Add hosts</span>
-                      </Button>
-                    )}
-                </div>
-              </div>
-              {renderActiveFilterBlock()}
-              {renderNoEnrollSecretBanner()}
-              {renderTable()}
+        <div className={`${baseClass}`}>
```

Comment:
you can place this classname on `MainContent` component. This removes the need to have this wrapping div

Reply:
No can do, MainContent will error out with JSX tag's 'children' prop expects a single child of type 'ReactChild', but multiple children were provided.

Reply:
ah right i forgot about that.